snake/main.py

```diff
@@ -189,6 +193,11 @@ def checkered_surface(screen: pygame.Surface) -> pygame.Surface:
 
     return checkered
 
+def score_update(score):
+    font = pygame.font.SysFont("sans-sarif", 35, True)
+    text = font.render("Score: " + str(score), 100, Color.WHITE) # Arguments are: text, anti-aliasing, color
+    pg_screen.blit(text, (650, 10))
```

Comment:
That blit line has a couple of issues. 
 The first is that you're referencing pg_screen, a variable that is from outer scope. It works now because pg_screen is defined when your function gets called, but you don't know if your code will still work in the future, if someone makes changes to pg_screen in the main code and then call your function. To keep that from happening, you could pass the screen surface to your function as a parameter, instead.
 The second is that you hard coded the position where the font will be drawn. A change in resolution would also break your code, making the score appear in the wrong position. To keep it in the upper right corner, you need to subtract the text's width from the screen's width, to get a relative position.